categories/software-development/rules-to-better-microservices.md

```diff
@@ -6,6 +6,7 @@ uri: rules-to-better-microservices
 index:
 - microservices
 - microservice-key-components
+- modular-monoliths
```

Comment:
its not a part of microservices, if anything it should be its own category with many rules showing how to do xyz in modular monolith architecture